tests/BitcoinGold/TWSignerTests.cpp

```diff
@@ -67,19 +67,18 @@ TEST(TWBitcoinGoldSigner, SignTransaction) {
 
     Data serialized;
     signedTx.encode(true, serialized);
-    // BitcoinGold Mainnet: https://btg2.trezor.io/tx/3e818ad25d73123b6c1c8099ed462aa5413a4ef57d66d9d260306c012753ba43
```

Comment:
since we change existing test, how to make sure the new test is accepted by network?

Reply:
Unclear how to verify.

Reply:
manually sign and broadcast a mainnet ltc or other cheaper coin

Reply:
Done, see test BitcoinSigning.SignLitecoinReal_8435